config/locales/bg/organisations.yml

```diff
@@ -15,62 +15,62 @@ bg:
           one:
           other:
         announcement:
-          one:
-          other:
+          one: Съобщение
+          other: Съобщения
```

Comment:
Hmm, we have both `one` and `other` here in commit 640580a62db62b89c7af80af12f38ad8207473f5, but we've removed the `one` key in other places, e.g. https://github.com/alphagov/collections/pull/2383/commits/640580a62db62b89c7af80af12f38ad8207473f5#diff-c8d38f9dfce6230fc33b107b1fb2f14406fac90137b008aeb55bb29d5382bacbL192.

Should we not be keeping the `one` key everywhere, even if we don't have values in all locales?

Reply:
Yes we should, if the language supports that form of pluralisation. The reason `one` has been deleted from certain locale files is because the language doesn't have an equivalent to the English `one`.

Work has been done prior to this to work out which pluralisation keys are actually needed for each language, you can see the decision to remove `one` from certain in locales in [this commit](https://github.com/alphagov/government-frontend/pull/2052/commits/2a81e71175cad844286188c9be5b2e8b8909c6be).